src/Yaml/ValidationPackagesConfig.fs

```diff
@@ -8,8 +8,8 @@ module ValidationPackagesConfig =
 
     let encoder (validationpackage : ValidationPackagesConfig) = 
         [
-            "validation_packages", Encode.resizearray ValidationPackage.encoder validationpackage.ValidationPackages
             Encode.tryInclude "arc_specification" Encode.string  (validationpackage.ARCSpecification)
```

Comment:
be better than us and put for example `"validation_packages"` as string Literal. I had to rerelease ARCtrl two times, because i missed updating one key.

Reply:
done, also re-used these literals as expected values in the related yaml tests